### PR TITLE
Add attestation templates

### PR DESCRIPTION
**Pull request by DevOnGekko**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 101-attestation-provider-create/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Create a Attestation Provider",
+  "description": "This template creates a Attestation Provider.",
+  "summary": "This template creates a Attestation Provider.",
+  "githubUsername": "devongekko",
+  "dateUpdated": "2018-10-04"
```
Comment (bmoore-msft) [inline 101-attestation-provider-create/metadata.json]: update the date

File: 101-attestation-provider-create/metadata.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Create a Attestation Provider",
+  "description": "This template creates a Attestation Provider.",
+  "summary": "This template creates a Attestation Provider.",
+  "githubUsername": "devongekko",
```
Comment (bmoore-msft) [inline 101-attestation-provider-create/metadata.json]: is this the proper owner?

File: 101-attestation-provider-create/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Create Attestation Provider
```
Comment (bmoore-msft) [inline 101-attestation-provider-create/README.md]: add a proper readme - what is this? why would I create one? is there documentation that can help me understand? is this GA?

File: 101-attestation-provider-create/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "attestationProviderName": {
+      "type": "string",
+      "metadata": {
```
Comment (bmoore-msft) [inline 101-attestation-provider-create/azuredeploy.json]: add a defaultValue - if it needs to be globally unique, use uniqueString()

File: 101-attestation-provider-create/metadata.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Create a Attestation Provider",
+  "description": "This template creates a Attestation Provider.",
```
Comment (bmoore-msft) [inline 101-attestation-provider-create/metadata.json]: add more detail here

File: 101-attestation-provider-create/README.md (hunk 3 above)
Comment (DevOnGekko) [reply 101-attestation-provider-create/README.md]: This service is currently only for preview and will be GA in Dec timeframe

**Pull request merged**